parser/parser_test.go

```diff
@@ -364,3 +364,18 @@ func TestIncompleteBlockComment(t *testing.T) {
 		}
 	}
 }
+
+func TestNilToken(t *testing.T) {
```

Comment:
I don't think people will understand what this test is about

Reply:
changing to this in #171 
```go
func TestInvalidToken(t *testing.T) {
	// This turns into the invalid token which in turn can't be found in prefix map,
	// it used to crash in earlier versions, this is the regression for that crash.
	// it also checks the error message in first line + first column case.
	inp := "@"
```

Reply:
OK 👍